### PR TITLE
Map link, text changes

### PR DESCRIPTION
**Pull request by saachipahwa**
- text changes: "allow" to "enable", got rid of "your" before "privacy", removed "Then,"
- increased text size on step goal page
- changed map link in intro screen
- added map link to settings page

### CODE REVIEW

**Review by thevirtuoso1973 — COMMENTED** [review]
@saachipahwa I believe I've used BASE_URL everywhere so the changed domain is not affected to the /map vs /mapDemo usage.

**Comment by thevirtuoso1973** [comment]
@saachipahwa i.e. so make the change.

**Review by thevirtuoso1973 — APPROVED** [review]

File: lib/pages/sharing_page.dart [hunk 1]
```diff
@@ -266,8 +266,7 @@ class SharingPageState extends State<SharingPage> {
                 RichText(
                     text: new TextSpan(children: [
                       new TextSpan(
-                          text:
-                              "Then, share the link to using one of these buttons:",
+                          text: "Share the link to using one of these buttons:",
```
Comment (thevirtuoso1973) [inline lib/pages/sharing_page.dart:269]: Wait why isn't this share the link **by** using one of these buttons

Comment (saachipahwa) [reply lib/pages/sharing_page.dart:269]: Joseph keeps asking for changes but I think we need to say it's a link to add the user.

**Comment by saachipahwa** [comment]
I'm going to make the text bigger on the set a step goal page

**Comment by saachipahwa** [comment]
the tutorial button is showing up in the AlertDialog for sharing wellbeing graph, even though it's set to false

**Comment by thevirtuoso1973** [comment]
Oh ye I must've just added the check before displaying coach mark, it's an ez fix.

**Comment by saachipahwa** [comment]
Should we have someplace on the settings page where users can access the map?

**Comment by thevirtuoso1973** [comment]
Yes we could put it below the Data & Privacy section that I added. So it serves as a demonstration of what their data is being used for.

**Comment by saachipahwa** [comment]
IMG_83DCF695193D-1

**Comment by saachipahwa** [comment]
IMG_1661

File: lib/pages/settings_page.dart [hunk 2]
```diff
@@ -78,7 +83,32 @@ class _SettingsPageState extends State<SettingsPage> {
               textAlign: TextAlign.center,
               style: Theme.of(context).textTheme.headline1),
           SizedBox(height: 10),
-          Flexible(child: settingsWidget)
+          Container(height: 330, width: 500, child: settingsWidget),
```
Comment (thevirtuoso1973) [inline lib/pages/settings_page.dart:86]: Prob not ideal to fix a height, so we'd prob have to change this height every time we add a setting thing.

**Review by thevirtuoso1973 — CHANGES_REQUESTED** [review]

**Review by thevirtuoso1973 — CHANGES_REQUESTED** [review]
Also Settings page doesn't look good on Android. Might need some spacing.

File: lib/pages/settings_page.dart (hunk 2 above)
Comment (saachipahwa) [reply lib/pages/settings_page.dart:86]: If we wrap it in flexible or expanded, it stretches to the whole screen and we'd need to add a sized box at the bottom to make the text visible. Should I do that instead?

**Comment by saachipahwa** [comment]
> Also Settings page doesn't look good on Android. Might need some spacing.

Spacing between what?

File: lib/pages/settings_page.dart (hunk 2 above)
Comment (thevirtuoso1973) [reply lib/pages/settings_page.dart:86]: No. There seems to be multiple ways to do it properly. 
Make them both expanded and modify the flex property appropriately. 
Or use spaceEvenly in the column's mainAxisAlignment property.

**Comment by saachipahwa** [comment]
Does it look better now

**Review by thevirtuoso1973 — APPROVED** [review]
LGTM

**Pull request merged**